definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -430,5 +430,96 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Yes"
+  },
```

Comment:
Generally speaking, I think we don't need to use any -nonprod.json files except for authorisation.
So you could put all the changes in the normal file as the event itself that displays them is not available anyway. This would make release to prod easier, would just need to change authorisation on the case event.

Reply:
Seems the npm test was not happy with me moving them from here to the normal file,  both CaseFields and CaseEventToFields entries are either in the normal or nonprod one. @Dizrupto Are you happy if this is left here as it still follows our Standards for this project.

Reply:
oow okay sure